Funky.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="Funky.css">
+</head>
+
+<body>
+    <h2> 
+    Time To Get Funky...
+    </h2>
+    <p>
+        <u>Flexbox Funkiness</u>
+    </p>
+    <button onclick="funkyRotate()">Let The Funkiness Commence</button>
+    <button onclick="funkyRotate2()">Let The Funkiness Commence</button>
+    <button onclick="funkyRotate3()">Let The Funkiness Commence</button>
+    <button onclick="funkyRotate4()">Let The Funkiness Commence</button>
+    <button onclick="funkyRotate5()">Let The Funkiness Commence</button>
+    <section id="FunkyArea">
+        <div class="funky">1</div>
+        <div class="funky">2</div>
+        <div class="funky">3</div>
+        <div class="funky">4</div>
+        <div class="funky">5</div>
+    </section>
+    <section id="FunkyArea2">

```

Comment:
Yep each of these sections could just be `class="FunkyArea"` unless you need to specifically target them via JS. And even then, you still could: `var funkyAreas = document.getElementsByClassName("FunkyArea")` Then you could target the 4th instance by: `var funkyArea4 = funkyAreas[3]` <- accessing the their index.

This is a really "repeat yourself" way of doing things. We like to keep things DRY.

Also a great usecase for React, but I won't go there ;)